ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -567,7 +581,13 @@ public boolean hasFailed() {
       }
       // if there is no possible way to use the remaining replicas to meet the success target,
       // deem the operation a failure.
-      return replicaInPoolOrFlightCount + replicaSuccessCount < replicaSuccessTarget;
+      // For Delete, even there is not enough quorum, if offline repair is enabled, continue to run it.
+      if (possibleRunOfflineRepair(replicaInPoolOrFlightCount + replicaSuccessCount)) {
```

Comment:
QQ: it seems like once the offline repair enabled, we will not fail the request. But will it be any chance that the record has not been successfully stored in the repair db? Or probably due to the availability is high, so we can ignore that?
Also, we probably want to have metrics to record how many times this issue happened?